Widescreen patch
Hoax - Dusk
Knight on left wall: prevent hiding based on camera offset
--- original
+++ modified
@@ -325,6 +325,7 @@
 call 294, 0x16, 0x1
 LABEL_50:
 yield
+jmp inl[:LABEL_50]
 call 257, stor[24], stor[25]
 jmp_cmp >, 0x5, stor[24], inl[:LABEL_51]
 call 106, 0x0
